Mathlib/Analysis/Normed/Field/Basic.lean

```diff
@@ -447,6 +447,21 @@ instance MulOpposite.instSeminormedRing : SeminormedRing αᵐᵒᵖ where
   __ := instRing
   __ := instNonUnitalSeminormedRing
 
+lemma norm_one_sub_mul (ha : ‖a‖ ≤ 1) : ‖c - a * b‖ ≤ ‖c - a‖ + ‖1 - b‖ :=
```

Comment:
I'm finding it hard to make sense of the name here. Maybe `norm_sub_mul`?

Reply:
Or `norm_sub_mul_le`

Reply:
Good point, corrected.